### PR TITLE
Added CoreDB for self hosted digital identities

### PR DESCRIPTION
**Pull request by rolandosborne**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,6 +208,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [Chatwoot](https://www.chatwoot.com) - Self-hosted customer communication platform, an alternative to Intercom & Zendesk. ([Source Code](https://github.com/chatwoot/chatwoot)) `MIT` `Ruby`
 - [Cherry](https://github.com/rafael-santiago/cherry) - Tiny webchat server. `GPL-2.0` `Go`
 - [Conduit](https://conduit.rs/) - A simple, fast, and reliable chat server powered by Matrix. ([Source Code](https://gitlab.com/famedly/conduit)) `Apache-2.0` `Rust`
+- [CoreDB](https://github.com/rolandosborne/CoreDB) - A decentralized network node for hosting your digital identity, supports the mobile app, IndiView, for sharing photos, videos and chatting. `Apache-2.0` `Javascript` `Java`
```
Comment (fbartels) [inline README.md:211]: Is there a specific reason to use your fork over the official repo?

Comment (Rmf77) [reply README.md:211]: Rey

Comment (Rmf77) [reply README.md:211]: > Is there a specific reason to use your fork over the official repo?

Comment (rolandosborne) [reply README.md:211]: Hi, I am the dev for the original project, but we will be closing it due to lack of traction and non-profit fees. I wanted to show the history of the project, so I forked it.

Comment (fbartels) [reply README.md:211]: Ah, now you have rewritten the git history and deleted the old orga. I personally would have made it the other way around, keep the github orga (after all they are free, so no need for a non-profit) and operate from there.

Comment (rolandosborne) [reply README.md:211]: I was worried the non-profit shutting down and the decentralized network being part of a separate GitHub would be confusing. You're probably right though, I should have left it alone.

**Review by nodiscc — CHANGES_REQUESTED** [review]
- The only supported installation method is Docker -> please change the language/platform to just `Docker`
- The provided [docker stack definition](https://github.com/rolandosborne/CoreDB/blob/main/tools/docker-stack.yaml) uses the [coredb/diatum](https://hub.docker.com/r/coredb/diatum) image which has not been updated for ~2 months and does not provide an auditable Dockerfile (I get a 404 error on this image's page on dockerhub). Please provide links to the full source code of the `coredb/diatum:latest` image

**Comment by nodiscc** [comment]
@rolandosborne ping, can you address the problems I mentioned above?

**Comment by rolandosborne** [comment]
I am rewriting the server in Golang, and will resubmit at that time. Sorry for the delayed response.

**Pull request closed without merging**